Life-Pill/pharmacy-pos-frontend
# Fixing and Enhancing Authentication Flow

## Pull Request: Fixing and Enhancing Authentication Flow

### Description
This pull request addresses several issues and enhancements related to the authentication flow within the application. It includes fixes for item update endpoint, order fetching functionality, and logout features for both managers and cashiers. Additionally, it introduces filtering capabilities for order fetching. The authentication flow has been thoroughly reviewed and completed, now including the removal of users.

### Changes Made
- Fixed item update endpoint
- Corrected order fetching functionality
- Implemented order fetching and filtering
- Added manager and cashier logout functionality
- Included temporary logout feature
- Completed authentication flow, including user removal

### Manual Testing Done
1. Tested item update functionality to ensure changes are properly reflected.
2. Verified order fetching and filtering, ensuring accurate results.
3. Tested manager and cashier logout functionalities to confirm proper operation.
4. Checked temporary logout feature for correctness.
5. Validated authentication flow, including user removal, to ensure seamless operation.


### Additional Information
This pull request has been thoroughly tested and reviewed to ensure the correctness and completeness of the implemented features.

---

Tharindu Jayawardhana  
Frontend Developer at LifePill
